### PR TITLE
[TASK] Use the Doctrine Symfony bundle

### PR DESCRIPTION
**Pull request by oliverklee**

### CODE REVIEW

**Comment by oliverklee** [comment]
Rebased and repushed.

File: Configuration/config.yml [hunk 1]
```diff
@@ -30,3 +30,25 @@ framework:
     assets: ~
     php_errors:
         log: true
+
+# Doctrine Configuration
```
Comment (samtuke) [inline Configuration/config.yml]: Some additional documentation here about whether these vars should be manually set would be useful, particularly their relationship to `parameters.yml` and Apache environment variables.

Comment (oliverklee) [reply Configuration/config.yml]: Done and re-pushed.

File: Configuration/config.yml [hunk 2]
```diff
@@ -30,3 +30,27 @@ framework:
...
+doctrine:
+    dbal:
+        # These variables come from parameters.yml. There, the values are read from environment variables
+        # and can also be set directly in the parameters.yml file.
```
Comment (oliverklee) [inline Configuration/config.yml]: @samtuke Like this …?

File: Configuration/parameters_template.yml [hunk 3]
```diff
@@ -1,4 +1,8 @@
 parameters:
+    # These variables are read from environment variables using the "env" construct.
+    # You can set environment variables in the Apache host configuration and also on the command line.
+    # If you cannot provide any environment variables, you can also set the variables in this file
+    # in the lines with "env(VARIABLE_NAME)".
```
Comment (oliverklee) [inline Configuration/parameters_template.yml]: @samtuke And like this …?

Comment (samtuke) [reply Configuration/parameters_template.yml]: Excellent, thanks!

**Pull request merged**